code/game/objects/items/flufflaunchers.dm

```diff
@@ -0,0 +1,251 @@
+/*		2077 WARCRIMES ARE NOW LEGAL
```

Comment:
This file should be moved to the `code/modules/projectiles/guns/ballistic` folder.